pkg/ocm/pullsecret_auth.go

```diff
@@ -33,61 +34,34 @@ func (a authentication) AuthenticatePullSecret(ctx context.Context, pullSecret s
 	}
 	defer connection.Close()
 
-	request := connection.Post()
-	request.Path("/api/accounts_mgmt/v1/token_authorization")
+	accessTokenAPI := connection.AccountsMgmt().V1()
+	request, err := amgmtv1.NewTokenAuthorizationRequest().AuthorizationToken(pullSecret).Build()
 
-	type TokenAuthorizationRequest struct {
-		AuthorizationToken string `json:"authorization_token"`
-	}
-
-	tokenAuthorizationRequest := TokenAuthorizationRequest{
-		AuthorizationToken: pullSecret,
-	}
-
-	var jsonData []byte
-	jsonData, err = json.Marshal(tokenAuthorizationRequest)
 	if err != nil {
 		return nil, err
 	}
-	request.Bytes(jsonData)
 
-	postResp, err := request.SendContext(ctx)
+	response, err := accessTokenAPI.TokenAuthorization().Post().Request(request).Send()
+
 	if err != nil {
 		return nil, err
 	}
 
-	if postResp.Status() != 200 {
+	if response.Status() != http.StatusOK {
```

Comment:
do we need to check the response status when using the sdk? iinm, it's already handled by Send(), i.e. err != nil

Reply:
I think that the error handling in the Send method is for umarshalling errors, not for the HTTP status itself.

Reply:
I will check this again. It was needed before the use of SDK.

Reply:
@danielerez you are right as usual. Removed it.